finish_release_test.py

```diff
@@ -134,3 +150,83 @@ async def test_set_release_date_no_file(test_repo_directory, timezone, mocker):
     await set_release_date("0.1.0", timezone, root=test_repo_directory)
     mock_check.assert_not_called()
     mock_output.assert_not_called()
+
+
+@pytest.mark.parametrize("has_require", [True, False])
+def test_update_go_mod(test_repo_directory, test_repo, has_require):
+    """update_go_mod should update and replace a go.mod file"""
+    go_mod_path = Path(test_repo_directory) / "go.mod"
+    contents = """module github.com/mitodl/ocw-course-hugo-starter
+
+go 1.13
+
+"""
```

Comment:
is this purposefully  broken down into multiple lines?

Reply:
Yes, so we can test the function on a file with the `require` line and one without